examples/index.html

```diff
@@ -65,35 +57,14 @@
         // mode: "distort", => default handle
         selected: true,
         corners: [
-          L.latLng(51.52, -0.1),
           L.latLng(51.52, -0.14),
-          L.latLng(51.5, -0.1),
-          L.latLng(51.5, -0.14)
+          L.latLng(51.52, -0.10),
+          L.latLng(51.50, -0.14),
+          L.latLng(51.50, -0.10)
         ],
         fullResolutionSrc: "large.jpg"
       }).addTo(map);
 
-      /* add custom icon functionality */
```

Comment:
@sashadev-sky Was this added in the readme? Maybe we should add this there and then remove this from here? Or do you think that this (removing this w/o adding anywhere) is okay?

Reply:
I uncommented it and it didn't work so I figured it was unfinished implementation! but there is an issue for documenting #194 if you want to take that up! This just isnt the place for it! I would do it myself but u know better what this function is. what do you think?